### ISSUE TITLE
Orgs: Migrate creation to sign up/sign in

### ISSUE THREAD

**Issue by JonEsparaz** [jesus-collective/mobile#345]
After #344, Organizations can be created, but via the dashboard. This needs to happen exclusively at sign up/sign in.

**Comment by JonEsparaz** [comment]
@GeorgeBellTMH for the situation where someone doesn't complete their profile on the first pass, we need a way to determine if the user is signing up as an individual or organization. Can you set up a custom attribute for this in Cognito? Or would it be better to save this data in DynamoDB?

**Comment by JonEsparaz** [comment]
We might also need another custom attribute for organization name since the mockups suggest that the user should fill out the organization name here:

**Comment by GeorgeBellTMH** [comment]
orgName, isOrg added...

**Comment by JonEsparaz** [comment]
@GeorgeBellTMH I think we need to edit the read/write permissions. I got the following error: "​A client attempted to write unauthorized attribute"

https://stackoverflow.com/questions/44013901/amazon-cognito-a-client-attempted-to-write-unauthorized-attribute

**Comment by GeorgeBellTMH** [comment]
I tried to add it them to the list in the json and it gave me an error. Need to read a bit more about it.

Sent from my iPhone

On Jul 29, 2020, at 7:34 PM, Jon Esparaz <notifications@github.com> wrote:

﻿

@GeorgeBellTMH<https://github.com/GeorgeBellTMH> I think we need to edit the read/write permissions. I got the following error: "​A client attempted to write unauthorized attribute"

https://stackoverflow.com/questions/44013901/amazon-cognito-a-client-attempted-to-write-unauthorized-attribute
